hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/metrics/ReadMetricsMessageTask.java

```diff
@@ -97,5 +97,11 @@ public String getMethodName() {
     public Object[] getParameters() {
         return new Object[] {parameters.uuid, parameters.fromSequence};
     }
+
+    @Override
+    public boolean isManagementTask() {
```

Comment:
I'm not sure if we need to mark "read metrics" task as an MC operation and restrict the access to it when trusted interfaces config option is in use. Potentially, we may use it elsewhere, say, build a Jet source that reads IMDG metrics.

@emre-aydin @blazember WDYT?

Reply:
I think we should be able to change it if we do use it elsewhere later but currently it is used by MC only and I don't see a reason to make it a non-management task. Just my 2c. I wouldn't object to making it a non-management task either.

Reply:
It could be that my understanding of the requirements is wrong, but isn't the differentiation between the connections and the private APIs in the requirements is to treat this task differently than the other MC tasks? (read: shouldn't this method return `false`?)

Reply:
The thing is that MC connection (the one with `MCJVM` connection type) is not not reliable marker for security restrictions. That's why this PR implements restrictions on per operation basis.

As for marking "read metrics" task as a non-MC one (thus, removing the whitelist restriction), I don't see this clearly stated in requirements. But as it may make some sense in future, I decided to start this discussion.

Reply:
I think, we can split it in the future when the need comes. For now, I would leave the metrics-reading as a management task as it's not available in our official clients.

Reply:
Sounds good. Marking this one as resolved.